server_test.go

```diff
@@ -3597,6 +3597,50 @@ func TestShutdownCloseIdleConns(t *testing.T) {
 	}
 }
 
+func TestShutdownWithContext(t *testing.T) {
+	t.Parallel()
+
+	ln := fasthttputil.NewInmemoryListener()
+	s := &Server{
+		Handler: func(ctx *RequestCtx) {
+			ctx.Success("aaa/bbb", []byte("real response"))
```

Comment:
Can you add a `time.Sleep(time.Millisecond*100)` here and test that `ShutdownWithContext` returns correctly after the timeout? The test you have now just tests the code that is similar to what `TestShutdown` already tests. Thanks!

Reply:
thanks